AEPServices/Sources/ServiceProvider.swift

```diff
@@ -114,15 +113,10 @@ public class ServiceProvider {
         }
     }
 
-    public var uiService: UIServiceInterface {
+    public var messageMonitor: MessageMonitor {
```

Comment:
nit: Can we keep the pattern of `service`  being the suffix for the variable name and the type?